LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -778,42 +784,65 @@ - (void)updateViewConstraints {
     
     if (self.isSimple) {
         
-        _simplePasscodeView.frame = CGRectMake(0, 0, _digitsCount * (kPasscodeCharWidth + _horizontalGap/2) - _horizontalGap/2 + _digitsCount/2, kPasscodeCharHeight);
-
         for (int i = 0; i < _digitsCount; i++) {
             
-            CGFloat constant = i == 0 ? 0 : i * _horizontalGap;
+            CGFloat constant = [self calculateConstantForPasscodeDigit:i];
```

Comment:
There are a ton of methods already, and this seems pretty short, could we inline it, please? After that, could you please rebase with master (it has +5 commits), and add and the change to `3.7.11` inside `CHANGELOG.md`, by also specifying the issue it closes, please?

I downloaded it, and otherwise it looks really great!
```objc
CGFloat constant = i == 0 ? 0 : _horizontalGap;
```